mon-pix/tests/unit/controllers/campaigns/skill-review-test.js

```diff
@@ -17,20 +18,57 @@ describe('Unit | Controller | Campaigns | Skill Review', function() {
       campaignParticipation: {
         isShared: false,
         set: setStub,
-        save: sinon.stub().resolves({})
+        save: sinon.stub().resolves({}),
+        campaignParticipationResult: {
+          masteryPercentage: 50,
+        }
       },
       assessment: {
         id: 'assessmentId',
         get: sinon.stub().withArgs('id').resolves('assessmentId'),
         state: 'completed',
         save: sinon.stub().resolves({}),
         codeCampaign: 'codeCampaign'
+      },
+      campaign: {
+        targetProfile: {
+          name: 'Cléa Numérique',
+        }
       }
     });
     controller.set('showButtonToShareResult', true);
     controller.transitionToRoute = sinon.stub();
   });
 
+  describe('#shouldShowPixEmploiBadge', () => {
+    it('should return true when user masterise more than 85 percent for pixEmploi profile ', function() {
```

Comment:
should return true when user **masters** more than 85 percent for pixEmploi profile